packages/@atjson/source-html/test/__snapshots__/table.test.ts.snap

```diff
@@ -674,77 +678,77 @@ exports[`Table valid tables converts column alignments 1`] = `
       "attributes": {
         "records": [
           {
-            "age": {
+            "age (2)": {
               "jsonValue": "20",
               "slice": "M00000007",
             },
-            "job": {
+            "job (3)": {
               "jsonValue": "fighter",
               "slice": "M00000008",
             },
-            "name": {
+            "name (1)": {
               "jsonValue": "laios",
               "slice": "M00000006",
             },
-            "notes": {
+            "notes (4)": {
               "jsonValue": "A strange but earnest person. He
         really really likes monsters",
               "slice": "M00000009",
             },
           },
           {
-            "age": {
+            "age (2)": {
```

Comment:
@colin-alexa so unfortuntately these fields need to be valid names for a SQL database since they are used by a databricks schema 😭

Reply:
lol really? it's user-generated content, don't we need to like....sanitize it then?

Reply:
we discussed irl and there's some silly things with how databricks works that is not terribly consistent with actual postgres rules.

Reply:
I'll try to raise this issue separately with the relevant team at conde to make this work